.github/workflows/platform-zxc-compile-platform-code.yaml

```diff
@@ -203,7 +203,7 @@ jobs:
         if: ${{ inputs.enable-unit-tests && steps.gradle-build.conclusion == 'success' && !cancelled() && always() }}
         with:
           gradle-version: ${{ inputs.gradle-version }}
-          arguments: -p platform-sdk check hammerTest --scan
+          arguments: -p platform-sdk check hammerTest --continue --scan
```

Comment:
`--continue` to grab all failing tests in one run (align with 'hedera-node' check run)